doc/modules/mixture.rst

```diff
@@ -152,35 +154,54 @@ priors) instead of data likelihood. The principle behind
 variational methods is the same as expectation-maximization (that is
 both are iterative algorithms that alternate between finding the
 probabilities for each point to be generated by each mixture and
-fitting the mixtures to these assigned points), but variational
+fitting the mixture to these assigned points), but variational
 methods add regularization by integrating information from prior
 distributions. This avoids the singularities often found in
 expectation-maximization solutions but introduces some subtle biases
 to the model. Inference is often notably slower, but not usually as
 much so as to render usage unpractical.
 
-Due to its Bayesian nature, the variational algorithm needs more
-hyper-parameters than expectation-maximization, the most
-important of these being the concentration parameter ``dirichlet_concentration_prior``. Specifying
-a high value of prior of the dirichlet concentration leads more often to uniformly-sized mixture
-components, while specifying small (between 0 and 1) values will lead
-to some mixture components getting almost all the points while most
-mixture components will be centered on just a few of the remaining
-points.
-
-.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_bayesian_gaussian_mixture_001.png
-   :target: ../auto_examples/mixture/plot_bayesian_gaussian_mixture.html
+Due to its Bayesian nature, the variational algorithm needs more hyper-
+parameters than expectation-maximization, the most important of these being the
+concentration parameter ``weight_concentration_prior``. Specifying a high value
+of prior of the dirichlet concentration leads more often to uniformly-sized
+mixture components, while specifying small (between 0 and 1) values will lead
+to some mixture components getting almost all the points while most mixture
+components will be centered on just a few of the remaining points.
+Moreover, the parameter ``weight_concentration_prior_type`` proposes two different priors
+to model the weight : a Dirichlet distribution or a Dirichlet process.
+
+The next figures compare the results obtained from the
+:class:`BayesianGaussianMixture` class models with a Dirichlet distribution
+(``weight_concentration_prior_type='dirichlet_distribution'``) for different values of
+``weight_concentration_prior``. Here, we can see the the value of the
+``weight_concentration_prior`` parameter has a strong impact on the effective
+number of active components obtained.
+
```

Comment:
We can also notice that large values for the concentration weight prior lead to more uniform weights when the type of prior is `'dirichlet_distribution'` while this is not necessarily the case for the `'dirichlet_process'` type (used by default).